R/compare_df_types.R

```diff
@@ -0,0 +1,245 @@
+#' Generate a comparison of data.frames (or similar objects) that indicates if
+#' they will successfully bind together by rows.
+#'
+#' @details Due to the returned "column_name" column, no input data.frame may be
+#'   named "column_name".
+#'
+#' @param ... data.frames or similar objects (such as tibbles), or lists of
+#'   data.frames (or similar objects).  The values may optionally be named
+#'   arguments; if named, the output column will be the name; if not named, the
+#'   output columne will be the data.frame name (see examples section).
+#' @param return Should a summary of "all" columns be returned, only return
+#'   "match"ing columns, or only "mismatch"ing columns?
+#' @param bind_check What method of binding should be used to determine matches?
```

Comment:
I prefer `binding` or maybe `bind_method` for the name of this argument - they're specifying the binding method, not the check - what do you think?  Sorry this is such a picky thing to point out 😬 , but b/c of the `...` argument this arg has to be named so does matter a bit.

Reply:
Improving the clarity of naming is always appreciated!  I used `bind_method` as that seems clearer to me.